clippy_lints/src/unwrap.rs

```diff
@@ -231,3 +236,7 @@ impl<'tcx> LateLintPass<'tcx> for Unwrap {
         walk_fn(&mut v, kind, decl, body.id(), span, fn_id);
     }
 }
+
+fn is_test_module_or_function(item: &Item<'_>) -> bool {
+    matches!(item.kind, ItemKind::Mod(..)) && item.ident.name.as_str().contains("test")
+}
```

Comment:
This function should already exist in the `clippy_lints::wildcard_imports` module. Can you move it to `utils` and reuse it here, please?

Reply:
Ok, thanks.